lerna.json

```diff
@@ -7,5 +7,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.0.0"
-}
\ No newline at end of file
+  "version": "0.8.0-dev.2"
```

Comment:
For now I have aligned all version numbers to agree with each other. We don't have to stay like this, in fact I imagine we won't once we go to version 1.0.0.